simulariumio/tests/converters/test_trajectory_converter.py

```diff
@@ -59,1011 +62,1180 @@ def mixed_agents_wrong_display_type2():
 
 
 def mixed_agents_invalid_agent_id():
+    # Agent IDs are larger than a 32 bit int can represent
     result = mixed_agents()
     result.agent_data.unique_ids[0][0] = MAX_AGENT_ID + 1
     return result
 
 
+# 3 default agents (radius 5-10) at given positions for 3 frames,
+# test string for plots
+default_agents_trajectory = three_default_agents()
+default_agents_converter = TrajectoryConverter(default_agents_trajectory)
+default_agents_data = JsonWriter.format_trajectory_data(default_agents_converter._data)
```

Comment:
Another thought, but may be for another PR: 
This is supposed to be testing the `TrajectoryConverter`, but I *think* a lot of the resulting data is handled by the `format_trajectory_data`. While these tests are nice to have, we should have unit tests that actually cover the `TrajectoryConverter` methods without involving any other classes